### PR TITLE
Add blog and video content to examples folder

### PR DESCRIPTION
**Pull request by jackos**
Gives content producers a place to link to blog and video content

### CODE REVIEW

File: .gitpod.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+  # List the start-up tasks. Learn more: https://www.gitpod.io/docs/configure/workspaces/tasks
+  tasks:
+    - name: Install Modular & Mojo
+      before: |
+        eval $(command gp env -e)
+      command: |
+        curl https://get.modular.com | MODULAR_AUTH=$MODULAR_AUTH sh -
```
Comment (dualvtable) [inline .gitpod.yaml:7]: How is `$MODULAR_AUTH` set and what is the default value?

File: examples/blogs-videos/tensorutils/__init__.mojo
Comment (dualvtable) [inline examples/blogs-videos/tensorutils/__init__.mojo:1]: Can you please add a copyright and license header? We want to release code under the Apache 2.0 license with the LLVM exception. You can grab the copyright/license header blurb from here:

https://raw.githubusercontent.com/modularml/mojo/main/examples/deviceinfo.mojo

File: examples/blogs-videos/tensorutils/tensorutils.mojo
Comment (dualvtable) [inline examples/blogs-videos/tensorutils/tensorutils.mojo:1]: Can you please add a copyright and license header? We want to release code under the Apache 2.0 license with the LLVM exception. You can grab the copyright/license header blurb from here:

https://raw.githubusercontent.com/modularml/mojo/main/examples/deviceinfo.mojo

**Comment by dualvtable** [comment]
LGTM - thanks! I suggested that we add a copyright/license header to each source code file that we release externally.

File: .gitpod.yaml (hunk 1 above)
Comment (jackos) [reply .gitpod.yaml]: This file wasn't meant to be there, removed thanks

**Pull request merged**